miso-web/src/main/webapp/scripts/list.js

```diff
@@ -33,6 +33,208 @@ ListUtils = (function() {
+    },
+    "institute": {
+      term: "institute:NAME",
+      help: "Check if the item came from the institute mentioned.  This is a partial match."
+    },
+    "external": {
+      term: "external:NAME",
+      help: "Checks if an item came from the external identifier or external name."
+    },
+    "box": {
+      term: "box:NAME",
+      help: "Checks if an item is located in a particular box.  The name can either be the partial name or partial alias of the box."
+    }
+  };
+
+  var makeTooltipHelp = function(target) {
+    return "Search syntax: \
+      <br/> \
+      <br/>"
+      + target.searchTermSelector(searchTerms).map(function(term) {
+        return term["term"];
+      }).join('<br/>');
+  };
+
+  var makeSearchTooltip = function(searchDivId, target) {
+    var searchTooltipId = "searchHelpTooltip_" + target.name.replace(/\s/g, '');
+
+    jQuery("#" + searchDivId).append(' \
+      <div id="' + searchTooltipId + '" class="tooltip"> \
+        <span> \
+          <img id="searchHelpQuestionMark" src="/styles/images/question_mark.png" \
+        </span> \
+        <span class="tooltiptext"> \
+          ' + makeTooltipHelp(target) + ' \
+        </span> \
+      </div>');
+
+    return searchTooltipId;
+  };
+
+  var makePopupTableBody = function(target) {
+    var result = "";
+    var targetTerms = target.searchTermSelector(searchTerms);
+
+    for (var i = 0; i < targetTerms.length; ++i) {
+      result += "<tr><td>" + targetTerms[i]["term"] + "</td><td>" + targetTerms[i]["help"] + "</td></tr>";
+    }
+
+    return result;
+  };
+
+  var makePopupTable = function(target) {
+    return ' \
+      <table class="searchHelpTable">\
+        <caption><h2>Search Terms</h2></caption>\
+        <thead> \
+          <tr><th>Syntax</th><th>Meaning</th></tr> \
+        </thead> \
+      ' + makePopupTableBody(target) + ' \
+      </table>';
+  };
+
+  var dateGrammar = '\
+    <table class="searchHelpTable"> \
+      <caption><h2>DATE Format</h2></caption> \
+      <thead> \
+        <tr><th>Format</th><th>Behaviour</th></tr> \
+      </thead> \
+      <tr><td>lasthour</td><td>Filter from 1 hour ago to the current time.</td></tr> \
+      <tr><td>today</td><td>Anything that happened on the current calendar day.</td></tr> \
+      <tr><td>yesterday</td><td>Filter for anything on the last calendar day.</td></tr> \
+      <tr><td>thisweek</td><td>Filter from Monday 00:00:00 of the current week to the present time.</td></tr> \
+      <tr><td>lastweek</td><td>Filter from Monday 00:00:00 of the previous week to Sunday 23:59:59 of the previous week.</td></tr> \
+      <tr><td><i>N</i>hours</td><td>Filter for anything from the current time to <i>N</i> hours ago.</td></tr> \
+      <tr><td><i>N</i>days</td><td>Filter for anything from the current time to <i>N</i>*24 hours ago.</td></tr> \
+      <tr><td>YYYY-MM-DD</td><td>Search from YYYY-MM-DD 00:00:00 to YYYY-MM-DD 23:59:59</td></tr> \
+    </table>';
+
+  var userGrammar = '\
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use string concatenation rather than line continuation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AMultilineStringLiterals)